tools/build-webview

```diff
@@ -68,9 +68,10 @@ while (( $# )); do
         --destination)
             # note: this doesn't permit an equals sign after `--destination`
             shift; dest="$1"; shift;;
-        --no-sanity-checks)
-            # disables target-directory sanity checks, for testing's sake
-            shift; sanity_checks=0;;
+        --script)
```

Comment:
It's a good idea at its core; but `--script` is not a meaningful name, even with the documentation added in the following commit. 

(Why not just `--sanity-checks`?)

Reply:
Yeah, I don't love the name. I wanted something non-specific about *what* the checks are, basically a name that summarizes the description in the new comment / in the subsequent usage message, but "script" isn't very clear in doing that.

I didn't go for `--sanity-checks` because I don't like the implication there's something wild or wrong about a different destination path. In particular when you ask some code to do something in conditions that it has "sanity checks" that would reject if you enabled them, that feels like the code is free to do wacky things that don't correspond to what it would do in proper conditions. Here, that would undermine the usefulness of being able to pass a convenient nonce directory and inspect the output there.

How about `--check-path-name`? Revised to that. (I like "name" a bit better than just `--check-path` or `--check-dest`, because either of the latter sound like they might check things like what's *inside* the dest directory.)